cryptkeeper_test.go

```diff
@@ -116,7 +117,8 @@ func TestCryptString(t *testing.T) {
 		if err != nil {
 			t.Fatalf("SetCryptKey should be valid, got: '%s'", err)
 		}
-		cs := CryptString{String: "another secret text"}
```

Comment:
this can stay what it was with, `Valid: true`, similar to how you made the change in the previous code.